doc/user_manual/existing_interfaces.tex

```diff
@@ -103,174 +103,6 @@ \subsection{Generic Interface}
   \item  \texttt{|}, the value that is going to be replaced by the Generic Interface, will be left- justified with a string length of ``10'';
 \end{itemize}
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%%%%% RAVEN  INTERFACE  (RAVEN running RAVEN) %%%%%%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{RAVEN Interface}
```

Comment:
@rouxpn I do not know how to say this...but you removed the RAVEN INTERFACE section... the existing_interface.tex file you are using is OLD.